src/Components/Employee.js

```diff
@@ -1,15 +1,49 @@
 import PetList from "./PetList";
 import "./Employee.css";
+import React, { useState, useEffect } from "react";
 
 export const Employee = () => {
-  return (
-    <article className="employee">
-      <h3>Staff Member Name</h3>
-      <h4>Staff Member Title</h4>
-      <button>Show Pets</button>
-      <PetList />
-    </article>
-  );
+  let empList = [];
+  const [currEmployees, setEmployees] = useState([]);
+  const [cid, setID] = useState([]);
+  const [toggle, setToggle] = useState(false);
+
+  useEffect(() => {
+    fetch("https://vet-resource-api-9-2.herokuapp.com/api/employees", {
+      method: "GET",
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setEmployees(data);
+      })
+      .catch((error) => {
+        console.error("Error:", error);
+      });
+  }, []);
+
+  empList = currEmployees.map((employee) => {
+    const { firstName, lastName, prefix, postfix, title, id } = employee;
+    let name = `${prefix || ""} ${firstName} ${lastName}${
```

Comment:
Very minor thing, but if there's no prefix, then there'll be an extra space in front of the name

Reply:
Could have an easy fix for this by doing .trim at the end of the string